### DISCUSSION TITLE
Switch sizes to mobile first

### DISCUSSION THREAD

**Discussion by marcelxpfeifer** [nuxt/image#697] [Polls]
I know this would be a major braking change, but I always find my self confused when defining sizes for images. Especially because the syntax is very similar to tailwind. Also it's best practice to create website in a mobile first approach.

Maybe we could add an option that allows generating it the other way around. This way a large code base would only need to toggle this option without rewriting all sizes. This would minimize the effort it takes to fix this breaking change.

**Comment by sirSaid** [comment]
您好，您发的邮件我已收到，谢谢。
